SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslator/LBStrategySpy.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
I think according to SpatialGDKTestGuidelines.h this would be a Stub implementation. Also, it's a generic enough implementation that I think it should live in the SpatialGDKTests/SpatialGDK/LoadBalancing so it can be reused by other code.

Reply:
Based on my realizations about the AbstractLBStrategy, I think this is functioning as a spy, but only because the base class is written that way, so I do think it should be a Stub.